### PR TITLE
BUGFIX: RAIL-2021 Fix Attribute Filter dropdown button title, BUGFIX: RAIL-2085 Fix Attribute Filter icon render issue

### PR DESCRIPTION
**Pull request by Tulean**
[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2872
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2612

# Related Jira tasks

### CODE REVIEW

**Comment by Tulean** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6633/ : SUCCESS in 7m 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2465/ : SUCCESS in 3s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3456/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2466/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2845/ : SUCCESS in 18m 25s

**Comment by Tulean** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1472.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2470/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2847/ : SUCCESS in 25m 26s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx [hunk 1]
```diff
@@ -21,8 +22,8 @@ const maxDelay = 2000;
 describe("AttributeDropdown", () => {
     function renderComponent(props: any = {}) {
         const {
-            projectId = "storybook",
-            onApply = (f: (...params: any[]) => any /* TODO: make the types more specific (FET-282) */) => f,
+            projectId = "projectId",
```
Comment (no23reason) [inline src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx]: Why the changes in the project id?

File: src/components/filters/AttributeFilter/tests/AttributeFilter.spec.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ describe("AttributeFilter", () => {
         };
 
         const props = {
-            projectId: "storybook",
+            projectId: "projectId",
```
Comment (no23reason) [inline src/components/filters/AttributeFilter/tests/AttributeFilter.spec.tsx]: Why the changes in the project id?

File: src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx (hunk 1 above)
Comment (Tulean) [reply src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx]: Because we used projectId everywhere in the links, so to be consistent with it I changed it.

Comment (no23reason) [reply src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx]: Ok :)

**Comment by no23reason** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6642/ : SUCCESS in 18m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2477/ : SUCCESS in 7s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3460/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2478/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2849/ : SUCCESS in 22m 11s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3461/ : SUCCESS in 18m 30s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6643/ : SUCCESS in 12m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1605/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2308/ : SUCCESS in 29m 30s
